### DISCUSSION TITLE
Dynamic and static path on same level

### DISCUSSION THREAD

**Discussion by andriiradkevych** [freshframework/fresh#2221] [General]
Is it possible to make routing with exceptions , I want to put `routes/[slug].tsx` and `routes/demo.tsx` on same level , but I see that it goest to `[slug].tsx` first . How to make it work , like first it check static pages `about, demo` etc , and then check dynamic `[slug].tsx` ?

**Comment by marvinhagemeister** [comment]
That's the way Fresh already works. It sorts routes by specificity. Static routes are matched first before dynamic ones. If you ever encounter a scenario where this is not the case, that would be a bug.
